airflow/models/asset.py

```diff
@@ -97,6 +98,24 @@ def resolve_ref_to_asset(
     return session.scalar(stmt)
 
 
+def remove_references_to_deleted_dags(session: Session):
```

Comment:
It would be great to add a test case for this method.

Reply:
Added